frontend/simple/views/components/Voting/Voting.vue

```diff
@@ -130,36 +159,121 @@ export default {
   },
   data () {
     return {
-      closeProposalState: {
-        state: null,
-        countdown: null
+      ephemeral: {
+        errorState: null,
+        closeCountdown: {
+          state: null,
+          countdown: null
+        }
       }
     }
   },
   computed: {
+    buttonText () {
+      const { value, originalValue } = this
+      if (this.type === TypeInvitation) {
+        return {
+          for: template(L('Invite {value}'), { value }),
+          against: L('Don\'t invite')
```

Comment:
Please use double-quotes in `L` functions if the string contains a single-quote. This is to ensure that our string scraper utility works correctly